site/src/pages/WorkspacePage/WorkspacePage.tsx

```diff
@@ -128,11 +127,11 @@ export const WorkspacePage: FC = () => {
           handleDelete={() => workspaceSend("ASK_DELETE")}
           handleUpdate={() => workspaceSend("UPDATE")}
           handleCancel={() => workspaceSend("CANCEL")}
-          resources={resources}
+          resources={workspace.latest_build.resources}
           builds={builds}
           canUpdateWorkspace={canUpdateWorkspace}
           workspaceErrors={{
-            [WorkspaceErrors.GET_RESOURCES_ERROR]: getResourcesError,
+            [WorkspaceErrors.GET_RESOURCES_ERROR]: refreshWorkspaceWarning,
```

Comment:
maybe rename `GET_RESOURCES_ERROR`?

Reply:
I will! Going to fix that in my workspace page error overhaul. Trying to limit this PR to xstate mostly if that's ok.